test/unit/auditor_test.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+let chai = require("chai");
+let sinon = require("sinon");
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'require' is not defined.